### PR TITLE
off canvas menu changed to overlay

### PR DESCRIPTION
**Pull request by 34cnk34**

### CODE REVIEW

File: src/assets/scss/_editontime.scss [hunk 1]
```diff
@@ -1,3 +1,14 @@
+/* off-canvas customization */
+.js-off-canvas-overlay {
+    background: rgba(255, 255, 255, 0.39)! important;
```
Comment (halukugurlu) [inline src/assets/scss/_editontime.scss]: please use !important, remove spacing.

File: src/assets/scss/_editontime.scss [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+}
+
+.off-canvas.is-transition-overlap.is-open {
+    z-index: 999! important;
```
Comment (halukugurlu) [inline src/assets/scss/_editontime.scss]: please use !important, remove spacing.

File: src/assets/scss/_editontime.scss (hunk 1 above)
Comment (34cnk34) [reply src/assets/scss/_editontime.scss]: ok thanks for the review

**Comment by halukugurlu** [comment]
good to go. Please merge

**Pull request merged**